js/assessment.js

```diff
@@ -197,6 +197,7 @@ define([
       var states = {};
       for (var i = 0, l = this._assessments.length; i < l; i++) {
         var assessmentModel = this._assessments[i];
+        if (!assessmentModel.get('_isAvailable')) continue;
```

Comment:
assessmentModel dosen't contains '_isAvailable'  property,
we need to first add it to assessmentModel

Reply:
> assessmentModel dosen't contains '_isAvailable' property

@akshay-bhavsar it should do... certainly seems to when I check it.

![image](https://user-images.githubusercontent.com/1249597/78554030-41f40400-7802-11ea-8b65-6e85b6700b95.png)

Reply:
@moloko  yes, i missed it.

my mistake!!

Reply:
That's the article model with the wrong variable name @akshay-bhavsar